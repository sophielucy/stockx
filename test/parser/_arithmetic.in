1+2;
4+5.6;

3-2;

2*3;

10/5;

9%4;

-100;

4 * (3-1);
