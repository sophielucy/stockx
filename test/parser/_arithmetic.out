Binop(IntLiteral(1), Add, IntLiteral(2))
Binop(IntLiteral(4), Add, FloatLiteral(5.6))
Binop(IntLiteral(3), Sub, IntLiteral(2))
Binop(IntLiteral(2), Mult, IntLiteral(3))
Binop(IntLiteral(10), Div, IntLiteral(5))
Binop(IntLiteral(9), Mod, IntLiteral(4))
Unop(Neg, IntLiteral(100))
Binop(IntLiteral(4), Mult, Binop(IntLiteral(3), Sub, IntLiteral(1)))
